addons/account/static/tests/helpers/mock_server.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { MockServer } from "@web/../tests/helpers/mock_server";
+
+patch(MockServer.prototype, {
+    async _performRPC(_route, { model, method, args }) {
+        if (model === "account.move" && method === "get_extra_print_items") {
+            return [];
+        }
+        return super._performRPC(...arguments);
+    },
+});
```

Comment:
do you need this if you add the mock in the AccountMove server model (file below)?

Reply:
I think it is still needed for legacy unit tests ?
https://runbot.odoo.com/runbot/build/65309350

Reply:
I thought your were modifying the new MockServer, not the legacy one. It is indeed necessary for legacy tests